### PR TITLE
Document the new modules and docs-link for all modules

### LINKED ISSUE #719: Add New Modules To The Docs
### Please describe the problem (or idea)

Some of the new modules which were added recently are not added to the `docs/modules.md` file.

- [ ] [Dither](https://github.com/publiclab/image-sequencer/blob/main/src/modules/Dither/info.json)
- [ ] [DrawRectangle](https://github.com/publiclab/image-sequencer/blob/main/src/modules/DrawRectangle/info.json)
- [ ] [PaintBucket](https://github.com/publiclab/image-sequencer/blob/main/src/modules/PaintBucket/info.json)
- [ ] [WhiteBalance](https://github.com/publiclab/image-sequencer/blob/main/src/modules/WhiteBalance/info.json)
- [ ] [Tint](https://github.com/publiclab/image-sequencer/blob/main/src/modules/Tint/info.json)
- [ ] [Threshold](https://github.com/publiclab/image-sequencer/blob/main/src/modules/Threshold/info.json)
- [ ] [FlipImage](https://github.com/publiclab/image-sequencer/blob/main/src/modules/FlipImage/info.json)

Current [modules.md](https://github.com/publiclab/image-sequencer/blob/main/docs/MODULES.md)

### Files to look into
- **info.json**: All the modules listed above link to their info.json files. You will get information about the modules there. You have to change these files to add a new property `docs-link` which is a link to their docs in the modules.md file.
- **modules.md**: The modules.md file linked above should be changed and docs to all modules should be added there.

#### Optional
All the module info.json files have linked directly to the modules.md file but the link does not link to the part that concerns the module.
https://github.com/publiclab/image-sequencer/blob/c445070937a9eb069745ced983f210b2f16da40b/src/modules/Channel/info.json#L12

Optionally add a link to the part of the modules.md file which documents the module.
For e.g. the following link links to the docs for channel module

https://github.com/publiclab/image-sequencer/blob/main/docs/MODULES.md#channel-module

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site.

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

### PR DESCRIPTION
**Pull request by lohitha02**
This commit documents new modules added to image-sequencer
- Dither
- Draw Rectangle
- Paint Bucket
- Tint
- Threshold
- WhiteBalance
- FlipImage

Fixes #719

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/is-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by lohitha02** [comment]
@HarshKhandeparkar please review it

**Comment by lohitha02** [comment]
@publiclab/is-reviewers

**Comment by HarshKhandeparkar** [comment]
Does this document FlipImage??

**Comment by lohitha02** [comment]
yes

File: src/modules/Rotate/info.json [hunk 1]
```diff
@@ -11,5 +11,5 @@
         "step": "1"
       }
     },
-    "docs-link":"https://github.com/publiclab/image-sequencer/blob/main/docs/MODULES.md"
+    "docs-link":"https://github.com/publiclab/image-sequencer/blob/main/docs/MODULES.md#Rotate-module"
```
Comment (HarshKhandeparkar) [inline src/modules/Rotate/info.json]:
```suggestion
    "docs-link":"https://github.com/publiclab/image-sequencer/blob/main/docs/MODULES.md#rotate-module"
```

File: src/modules/Saturation/info.json [hunk 2]
```diff
@@ -11,5 +11,5 @@
             "step": 0.1
         }
     },
-    "docs-link":"https://github.com/publiclab/image-sequencer/blob/main/docs/MODULES.md"
+    "docs-link":"https://github.com/publiclab/image-sequencer/blob/main/docs/MODULES.md#Saturation-module"
```
Comment (HarshKhandeparkar) [inline src/modules/Saturation/info.json]:
```suggestion
    "docs-link":"https://github.com/publiclab/image-sequencer/blob/main/docs/MODULES.md#saturation-module"
```

**Comment by lohitha02** [comment]
@HarshKhandeparkar anymore changes?

File: docs/MODULES.md [hunk 3]
```diff
@@ -6,30 +6,38 @@ List of Module Documentations
...
+4.  [Add QR](#Add-QR-module)
+5.  [Average](#average-module)
+6.  [Blend](#blend-module)
+7.  [Blur](#blur-module)
+8.  [Brightness](#brightness-module)
+9.  [Channel](#channel-module)
+10. [Colorbar](#colorbar-module)
+11. [Colormap](#colormap-module)
+12. [Contrast](#contrast-module)
+13. [Convolution](#convolutioon-module)
```
Comment (HarshKhandeparkar) [inline docs/MODULES.md]:
```suggestion
13. [Convolution](#convolution-module)
```
You can commit this suggestion directly from the browser. No need to make changes manually.

File: docs/MODULES.md [hunk 4]
```diff
@@ -6,30 +6,38 @@ List of Module Documentations
...
+14. [DecodeQr](#decodeQr-module)
+15. [Dither](#dither-module)
+16. [DrawRectangle](draw-rectangle-module)
```
Comment (HarshKhandeparkar) [inline docs/MODULES.md]:
```suggestion
16. [DrawRectangle](#draw-rectangle-module)
```
Here as well!

File: docs/MODULES.md [hunk 5]
```diff
@@ -6,30 +6,38 @@ List of Module Documentations
...
+19. [FlipImage](#flipimage-module)
+20. [Gamma-Correction](#gamma-correction-module)
+21. [Gradient](#gradient-module)
+22. [Histogram](#histogram-module)
+23. [Import-image](#import-image-module)
+24. [Invert](#invert-module)
+25. [Ndvi](#ndvi-module)
+26. [Ndvi-Colormap](#ndvi-colormap-module)
+27. [Overlay](#overlay-module)
+28. [PaintBucket](paint-bucket-module)
```
Comment (HarshKhandeparkar) [inline docs/MODULES.md]:
```suggestion
28. [PaintBucket](#paint-bucket-module)
```

File: src/modules/AddQR/info.json [hunk 6]
```diff
@@ -14,6 +14,6 @@
         "default": "https://github.com/publiclab/image-sequencer"
       }
     },
-    "docs-link":"https://github.com/publiclab/image-sequencer/blob/main/docs/MODULES.md"
+    "docs-link":"https://github.com/publiclab/image-sequencer/blob/main/docs/MODULES.md#add-qR-module"
```
Comment (HarshKhandeparkar) [inline src/modules/AddQR/info.json]:
```suggestion
    "docs-link":"https://github.com/publiclab/image-sequencer/blob/main/docs/MODULES.md#add-qr-module"
```

File: src/modules/NdviColormap/info.json [hunk 7]
```diff
@@ -2,5 +2,5 @@
     "name": "NDVI-Colormap",
     "description": "Sequentially Applies NDVI and Colormap steps",
     "inputs": {},
-    "docs-link": "https://github.com/publiclab/image-sequencer/blob/main/docs/MODULES.md"
-}
\ No newline at end of file
+    "docs-link": "https://github.com/publiclab/image-sequencer/blob/main/docs/MODULES.md#Ndvi-Colormap-module"
```
Comment (HarshKhandeparkar) [inline src/modules/NdviColormap/info.json]:
```suggestion
    "docs-link": "https://github.com/publiclab/image-sequencer/blob/main/docs/MODULES.md#ndvi-colormap-module"
```

Comment (Harshithpabbati) [reply src/modules/NdviColormap/info.json]: Make this change @lohitha02

**Comment by jywarren** [comment]
Oh wow. Fantastic work here!!!

**Pull request merged**